CHANGELOG.md

```diff
@@ -0,0 +1,318 @@
+# Fastlane CHANGELOG
+
+## Master
+
+* None.  
```

Comment:
Yes, so that section doesn't stay empty right after a release is done and looks ugly.

The `.` here is not just a full-stop, it's a full-stop followed by two spaces. That's what we use at the end of _every_ changelog entry, because Markdown interpret those specially, formatting the resulting rendered text as a line break within the same paragraph (as opposed to ignoring single CRLF used for wrapping, and as opposed to creating a new paragraph when you use multiple CRLFs)

Sure, For this special case of `None` we can definitely get rid of the full-stop + 2-spaces, as there's nothing on the next line, contrary to a typical changelog entry where it's followed by the links to the author & PR# which we want to keep on the next line but in the same paragraph. But keeping the full-stop + 2 spaces here keeps the convention everywhere, even for those "dummy" entries, and makes it consistent everywhere.